scripts/data-retention-remove-user-data.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""
+Our data retention policy is that users have accounts deactivated and personal data stripped after 3 years without
+a login.
+
+This script is very simple and has not been upgraded to accept any arguments to prevent the possibility of accidental
+deletion. If you are in doubt use the dry run option.
+
+Usage: data-retention-remove-user-data.py <stage> <api_token> [--dry-run] [--verbose]
+
+Options:
+    --stage=<stage>                                       Stage to target
+    --api_token=<api_token>                               API Token
+
+    --dry-run                                             List account that would have data stripped
+    --verbose
+    -h, --help                                            Show this screen
+
+Examples:
+    ./scripts/data-retention-remove-user-data.py preview myToken
+    ./scripts/data-retention-remove-user-data.py preview myToken --dry-run --verbose
+
+"""
+import logging
+import sys
+from datetime import datetime, timedelta
+from docopt import docopt
+
+from dmapiclient import DataAPIClient
+
+from dmscripts.helpers.env_helpers import get_api_endpoint_from_stage
+from dmscripts.helpers import logging_helpers
+
+sys.path.insert(0, '.')
+
+
+if __name__ == "__main__":
+    arguments = docopt(__doc__)
+
+    # Get script arguments
+    stage = arguments['<stage>']
+    api_token = arguments['<api_token>']
```

Comment:
@samuelhwilliams is changing how our scripts use API tokens - see https://github.com/alphagov/digitalmarketplace-scripts/pull/244. Let's use the helper instead.

Reply:
Niiiiiiicccceee!! Done 👍